aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -445,4 +448,9 @@ public void close() throws IOException {
   public void setConf(Configuration conf) {
     this.hadoopConf = conf;
   }
+
+  @Override
+  protected Map<String, String> properties() {
```

Comment:
[question] Looks like we are missing test cases for Glue catalog ?

Reply:
+1

Reply:
Agreed, we should have test cases for Glue catalog.

Reply:
I have added the tests for Glue catalog.